packages/ui-api/src/util/isTestChain.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017-2019 @polkadot/ui-api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { isUndefined } from '@polkadot/util';
+
+const re = new RegExp('(dev|loc)', 'i');
```

Comment:
Told @Stefie that I'm am not 100% happy with this.

So historically the chain was called `dev`, so we needed to check both `Development` and `dev` - I would adjust it now to be specific, `Development` & `Local` - exact match. (Need to adjust polkadot-js apps as well)

Reply:
Can't we put that check in a very tiny npm module, so we only need to maintain it once and are using the same condition everywhere?

Reply:
It fits into either `polkadot-js/common/util` or even as a new `polkadot-js/ui/ui-util` - going to move the utils from apps into one of the two. (There are a bundle that are actually general utilities)